src/knex/migrations/20220126135225_add_manager_column_for_vacuumlabs.js

```diff
@@ -0,0 +1,11 @@
+exports.up = async (knex) => {
```

Comment:
you could do a single migration updating both tables. just for the future, no action required.